### DISCUSSION TITLE
除了首页之后,没有其他的地方有坤元素了啊,请务必多在组件里面添加一些坤元素

### DISCUSSION THREAD

**Discussion by ljgx** [ikun-svelte/ikun-ui#163] [General]
感觉就首页有个坤元素,其他组件好像找不到坤元素的地方,ui整体是可以的,但是我想看的是坤元素,请多加坤元素在组件里面,形成我坤和组件融为一体那种感觉,不然只有首页有,内核没就不吸引人那

**Comment by baiwusanyu-c** [comment]
welcome pr

**Comment by like0413** [comment]
原来你想和我家坤坤合体，请问如何配置呢？可以看下连接器吗？

**Comment by haobarry** [comment]
哈哈 等我也pr个组件， 一定要含坤量高的

**Comment by KnightKilito** [comment]
附议

**Comment by kkkktigb** [comment]
坤元素是卖点，但是坤元素太少了

**Comment by leonardo110** [comment]
建议可以把loading动画做成打篮球或者左右晃动的经典画面
